### PR TITLE
No. More. Mosin.

### PR DESCRIPTION
**Pull request by ZeroNetAlpha**
## About The Pull Request

Pulls Mosin Nagants from the Trash and low tier gun spawning pools. This should technically remove them from most rounds for the time being but that's only until we can decide on where they really should go.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
del: Deleted the Mosin Nagant from the loot tables of trash piles and low tier guns.
/:cl:

### CODE REVIEW

**Comment by Faire-Fuzzy** [comment]
But why?

File: code/game/objects/effects/spawners/themed_loot_tables.dm [hunk 1]
```diff
@@ -179,7 +179,7 @@
 		/obj/item/gun/ballistic/rifle/hunting = 20,
 		/obj/item/gun/ballistic/revolver/widowmaker = 17,
 		/obj/item/gun/ballistic/shotgun/hunting = 10,
-		/obj/effect/spawner/bundle/f13/mosin = 8,
+		//obj/effect/spawner/bundle/f13/mosin = 8,
```
Comment (Superlagg) [inline code/game/objects/effects/spawners/themed_loot_tables.dm:182]: Please replace this with a hunting rifle or something.

**Review by Superlagg — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/spawners/f13lootdrop.dm [hunk 2]
```diff
@@ -818,7 +818,7 @@
 	name = "very low tier non-hobo ballistic gun"
 	loot = list(/obj/effect/spawner/bundle/f13/ninemil,
 				/obj/effect/spawner/bundle/f13/caravan_shotgun,
-				/obj/effect/spawner/bundle/f13/mosin,
+				//obj/effect/spawner/bundle/f13/mosin, No. More. Mosin.
```
Comment (Superlagg) [inline code/game/objects/effects/spawners/f13lootdrop.dm:821]: Dont need to remove it from the actual loot altogether. If that's no good, you can rename and reflavor the mosin to some kind of generic hunting rifle.

**Review by Superlagg — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/spawners/themed_loot_tables.dm (hunk 1 above)
Comment (Superlagg) [reply code/game/objects/effects/spawners/themed_loot_tables.dm:182]: That, or the pipe gun, or some kind of other gun, dont like just removing things.

**Comment by GremlingSS** [comment]
Im genuinely a fan of the mosin :( whyyy
Can't we just nerf it or something? why are we basically removing it from being able to get gotten'd?

File: code/game/objects/effects/spawners/themed_loot_tables.dm (hunk 1 above)
Comment (ZeroNetAlpha) [reply code/game/objects/effects/spawners/themed_loot_tables.dm]: Resolved, replaced with Hunting rifle as requested.

File: code/game/objects/effects/spawners/f13lootdrop.dm (hunk 2 above)
Comment (ZeroNetAlpha) [reply code/game/objects/effects/spawners/f13lootdrop.dm]: Generic hunting rifle was already in the game. I've replaced some references to mosin with the varmint rifle, others with the hunting rifle.

**Comment by ZeroNetAlpha** [comment]
The problem with the mosin is that it was way too easy to get 50 of them from trash piles in a round, something that should not have been possible.

**Review by Superlagg — APPROVED** [review]
Eh fair, we'll put the mosin somewhere.

**Pull request merged**